app/user/views.py

```diff
@@ -0,0 +1,48 @@
+import jwt
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework_simplejwt.tokens import RefreshToken
+
+from app.user.serializers import UserSerializer
+
+User = get_user_model()
+
+
+class UserRegister(APIView):
+    def post(self, request: Request, format: str = "json") -> Response:
+        serializer = UserSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        user = serializer.save()
+        refresh = RefreshToken.for_user(user)
+        response = serializer.data
+        response["refresh"] = str(refresh)
+        response["access"] = str(refresh.access_token)
+
+        return Response(response, status=status.HTTP_201_CREATED)
+
+
+class VerifyEmail(APIView):
+    def get(self, request: Request) -> Response:
+        token = request.GET.get("token")
+        try:
```

Comment:
You can have this in the serializers.py in VerifyEmailSerialier

Reply:
Changed